package/doc/sphinx/source/documentation_pages/references.rst

```diff
@@ -93,10 +93,7 @@ If you use the Path Similarity Analysis (PSA) code in
 If you use the implementation of the ENCORE ensemble analysis in
 :mod:`MDAnalysis.analysis.encore` please cite [Tiberti2015]_.
 
-.. [Tiberti2015] M. Tiberti, E. Papaleo, T. Bengtsen, W. Boomsma,
-   and K. Lindorff-Larsen. ENCORE: Software for quantitative ensemble
-   comparison. *PLoS Comput Biol*, **11** (2015), e1004415.  doi:
-   `10.1371/journal.pcbi.1004415`_
+[Tiberti2015]_
```

Comment:
Actually I like to have just one definition in the docs. Then we are guaranteed to always use the same.